projects/postgresql/sql.dict

```diff
@@ -0,0 +1,282 @@
+#
+# AFL dictionary for SQL
+# ----------------------
+#
+# Modeled based on SQLite documentation, contains some number of SQLite
```

Comment:
Was this updated to actually be PG-specific?  If so, this comment should be updated to reflect that.  If not, then it seems likely it would need to be...

Reply:
This is not PG-specific, more SQLite-specific. I thought PG might benefit from it but it can be taken out if not.